api-report/container-definitions.api.md

```diff
@@ -444,7 +444,9 @@ export interface IResolvedFluidCodeDetails extends IFluidCodeDetails {
 export interface IRuntime extends IDisposable {
     createSummary(blobRedirectTable?: Map<string, string>): ISummaryTree;
     getEntryPoint?(): Promise<FluidObject | undefined>;
-    getPendingLocalState(): unknown;
+    getPendingLocalState(props?: {
+        shutdownBlobUpload?: boolean;
```

Comment:
i don't think this naming is very clear. to me it implies that the upload of blobs will be shutdown.

Reply:
I'll remove the props object from the method params so naming is not relevant to the user I believe.

Reply:
it has to remain here. this is a the only place it should be

Reply:
well here and container runtime, it shouldn't be exposed in the loader layer

Reply:
basically, we want to minimally expose things, so only provide options where they are necessary and relavent to keep our surface area small and well structured